bundles/io/org.openhab.io.dropbox/README.md

```diff
@@ -1,19 +1,20 @@
 # Dropbox Synchronization Service
 
-This service will synchronize files on your openHAB server, such as configuration and log files, to and/or from your Dropbox account.
+This service will synchronize files on the openHAB server, such as
+configuration and log files, to and/or from a Dropbox account.
 
-The intended main use cases are backing up openHAB configuration and log files to a version-able cloud space and transporting changed files back to openHAB after editing them with the openHAB Designer on the administrator's desktop PC.
+The main use case is backing up openHAB configuration and log files
+to a version-able cloud space and transporting changed files back to openHAB
+after editing them with the openHAB Designer on the administrator's desktop PC.
 
-> NOTE: This service is currently disabled, due to [#4588](https://github.com/openhab/openhab1-addons/issues/4588).
 
 ## Service Configuration
 
 This service can be configured in the file `services/dropbox.cfg`.
 
 | Property | Default | Required | Description |
 |----------|---------|:--------:|-------------|
-| appkey   | `gbrwwfzvrw6a9uv` |  Yes due to [#4588](https://github.com/openhab/openhab1-addons/issues/4588) | the default app key is defunct and code changes are necessary. |
-| appsecret | `gu5v7lp1f5bbs07` | Yes due to [#4588](https://github.com/openhab/openhab1-addons/issues/4588) | the default app secret is defunct and code changes are necessary. |
+| personalAccessToken | | Yes   | This is the generated access token; see instructions below |
```

Comment:
It's great that all that will be needed after merge is a token, without needing negotiation steps!